tests/got/test_logging.js

```diff
@@ -1,55 +0,0 @@
-'use strict'
-
-const debug = require('debug')
-const sinon = require('sinon')
-const { expect } = require('chai')
-const nock = require('../..')
-const got = require('./got_client')
-
-describe('Logging using the `debug` package', () => {
-  let logFn
-  beforeEach(() => {
-    logFn = sinon.stub(debug, 'log')
-    debug.enable('nock*')
-  })
-
-  afterEach(() => {
-    debug.disable('nock*')
-  })
```

Comment:
is the lack of `.enable` and `.disable` the reason why we can't keep the tests? Couldn't we just set environment variables? I think we should keep log output tests, I usually like snapshot tests, but anything is better than nothing